site/js/ChannelsActor.js

```diff
@@ -0,0 +1,106 @@
+import Actor from './Actor.js'
+import { get, post } from './api.js'
+
+export default class ChannelsActor extends Actor {
+  init() {
+    this.on('update channel list', channels => {
+      // Join the first channel if we're not already in one
+      if (this.activeChannelID === null) {
+        const defaultChannel = channels[0]
+
+        if (defaultChannel)
+          this.viewChannel(defaultChannel.id)
```

Comment:
I'm of the One True Brace Style - always add braces around blocks, *even if* there's only one statement in the block. But I'm being nitpicky! :woman_shrugging: